dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala

```diff
@@ -33,53 +31,89 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
 import com.fasterxml.jackson.databind.SerializationFeature
+import za.co.absa.enceladus.dao.MenasRestDAO.log
 
 object EnceladusRestDAO extends EnceladusDAO {
   val conf = ConfigFactory.load()
   val restBase = conf.getString("menas.rest.uri")
   private val userName = LoggedInUserInfo.getUserName
 
+  var sessionCookie: String = ""
+  var csrfToken: String = ""
+
   private val log = LogManager.getLogger("enceladus.conformance.EnceladusRestDAO")
 
   val objectMapper = new ObjectMapper()
-      .registerModule(DefaultScalaModule)
-      .registerModule(new JavaTimeModule())
-      .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
-  
+    .registerModule(DefaultScalaModule)
+    .registerModule(new JavaTimeModule())
+    .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+
+  def postLogin(username: String, password: String) = {
+    try {
+      val httpClient = HttpClients.createDefault
+      val url = s"$restBase/login?username=$username&password=$password&submit=Login"
```

Comment:
Does this take into account special characters that users might have in their passwords?